src/state/GlobalStore.js

```diff
@@ -40,17 +40,17 @@ class GlobalStore {
   }
 
   addReducers() {
-    addReducer('empty', () => {
-      resetGlobal();
-      this.addCallbacks();
-      this.addReducers();
+    addReducer('empty', global => {
+      const { investigation } = global;
```

Comment:
`resetGlobal()` returns the global state to it's original value, which could be an invalid value loaded from local storage. Instead, delete the local storage and set a fresh empty state.